### PR TITLE
[projects/arm/] Extending rounding theorems to RTO

### PR DESCRIPTION
**Pull request by voskh0d**

### CODE REVIEW

**Comment by voskh0d** [comment]
Hi, I've sent a mail to David explaining those changes, could we wait for his approval before merging ? Thanks

File: books/projects/arm/utils/rtl-utils.lisp [hunk 1]
```diff
@@ -2326,6 +2338,21 @@
                                  (- (expo (spn f))))))
            :in-theory (enable drnd-ext expo))))
 
+(defthm drnd-ext-exactp-c
```
Comment (mayankmanj) [inline books/projects/arm/utils/rtl-utils.lisp:2341]: Could we disable this theorem? 

Otherwise, the changes look good to me.

**Comment by voskh0d** [comment]
Hey, sorry I did see your comment, I think I've disabled the mail notifications. I've done your change (it is indeed better), so I think we can merge this now Thanks :)

**Comment by MattKaufmann** [comment]
Merging at David Russinoff's request.

**Pull request merged**